uniconnect-epfl/uniconnect
# feat: add user events to profile

# What I did
 I added the event Screen on the user profile

# How I did it
I adapted the events Screen component from Home screen to be usable in profile screen
# How to verify it

Go to the profile screen and see events there
I added test that cover all new features

# Demo video

https://github.com/uniconnect-epfl/uniconnect/assets/108724823/b0ade128-49c4-4f8f-8699-85383a20bbfd


# Pre-merge checklist
The changes I have introduced:
